### ISSUE TITLE
In Firefox, keyboard cursor navigation gets trapped inside code block: CMU-290

### ISSUE THREAD

**Issue by darrensiegel** [Simon-Initiative/oli-torus#520]
On Chrome I can use the keyboard up and down arrow keys and traverse my cursor through all content on a page, including into and out of code blocks.  On Firefox, my cursor gets "trapped" in code blocks.  When I'm on the last line and I press "down" instead of the cursor going to the block following the code block, the cursor goes back to the top line.  Similarly, when I click "up" arrow at the top the cursor goes to the bottom line of the code block.

**Comment by LEDThereBeLight** [comment]
One note - using the left and right arrow keys will let you move through the codeblock on firefox. not sure what is preventing the up/down arrow keys from escaping the block

**Comment by LEDThereBeLight** [comment]
Still sort of an issue as of 3/1/22, but it's not clear what the fix should be. Arrows probably should navigate the code in a code block.
